src/components/EditorBody.vue

```diff
@@ -95,7 +95,7 @@ onMounted(() => {
     (highlight) => {
       const diagnostics: Diagnostic[] = []
       if (highlight) {
-        let lineI = highlight.line
+        let lineI = highlight.line + 1
```

Comment:
A file like this kept on causing errors, so I added this to account for codemirror's 1-indexing of lines.

![Screenshot 2025-01-19 at 2 35 59 AM](https://github.com/user-attachments/assets/9fd4030a-fe02-4578-84cf-f4452717870d)